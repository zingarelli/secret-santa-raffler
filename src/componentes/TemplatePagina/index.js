import { Outlet } from "react-router-dom";
import { RecoilRoot } from "recoil";
import Cabecalho from "../Cabecalho";
import Card from "../Card";

export default function TemplatePagina() {
    return (
        <RecoilRoot>
            <Cabecalho />
            <Card>
                <Outlet />
            </Card>
        </RecoilRoot>
    )
}