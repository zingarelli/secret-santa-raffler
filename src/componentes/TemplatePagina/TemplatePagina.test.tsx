import { render } from "@testing-library/react"
import TemplatePagina from ".";

describe('A pÃ¡gina de template', () => {
    test('deve ser renderizada corretamente', () => {
        const { container } = render(<TemplatePagina />);

        expect(container).toMatchSnapshot();
    });
})