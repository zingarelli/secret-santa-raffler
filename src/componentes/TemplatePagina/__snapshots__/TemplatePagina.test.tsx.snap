// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A p√°gina de template deve ser renderizada corretamente 1`] = `
<div>
  <header
    class="container"
  >
    <div
      aria-label="logo do sorteador de amigo secreto"
      class="logo"
      role="img"
    />
    <img
      alt="pessoa chacoalhando uma caixinha para sortear um amigo secreto"
      class="participante"
      src="/images/participante.png"
    />
  </header>
  <div
    class="card"
  />
</div>
`;
